code/src/components/summary.css

```diff
@@ -0,0 +1,52 @@
+.summary-section {
```

Comment:
The fact that you made the review display on a separate page is a really nice touch. Kind of neat for the user to be able to read the summary on a page of its own. I liked that. 👌